actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -794,4 +791,20 @@ def test_method_is_case_insensitive
     end
     assert_response :success
   end
+
+
+  def assert_presence_and_fetch_form_csrf_token
+    assert_select 'input[name="custom_authenticity_token"]' do |input|
+      form_csrf_token = input.first['value']
+      assert_not_nil form_csrf_token
+      return form_csrf_token
+    end
+  end
+
+  def assert_matches_session_token_on_server(form_token, method = 'post')
+    actual = @controller.send(:unmask_token, Base64.strict_decode64(form_token))
+    expected = @controller.send(:per_form_csrf_token, session, '/per_form_tokens/post_one', method)
+    assert_equal expected, actual
+  end
+
```

Comment:
Nitpick: can you remove this extra space here, and you have 2 spaces above where "assert_presence_and_fetch_form_csrf_token" starts. Can you remove those too?